pkgs/development/python-modules/bjoern/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi, libev, python }:
+
+buildPythonPackage rec {
+  pname = "bjoern";
+  version = "2.2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1w5z9agacci4shmkg9gh46ifj2a724rrgbykdv14830f7jq3dcmi";
+  };
+
+  buildInputs = [ libev ];
+
+  checkPhase = ''
+    ${python.interpreter} tests/keep-alive-behaviour.py 2>/dev/null
```

Comment:
I'm not sure it's wise to use `2>/dev/null`. Having some information when something goes wrong would be nice, wouldn't it?

Reply:
The proper way to silence these messages is to pipe the test's subprocesses' stderr to /dev/null, as suggested in the linked GitHub issue.